story/inputs.stories.js

```diff
@@ -1,28 +1,30 @@
 import { storiesOf } from '@storybook/html'; // eslint-disable-line import/no-extraneous-dependencies
 import { // eslint-disable-line import/no-extraneous-dependencies
-  withKnobs, radios,
+  withKnobs, radios, boolean,
 } from '@storybook/addon-knobs';
 
 const stories = storiesOf('Inputs', module);
 stories.addDecorator(withKnobs);
 
-stories.add('input.radio', () => `
-      <label>
-        <input type="radio" class="nes-radio" name="answer" checked />
-        <span>Yes</span>
-      </label> <label>
-        <input type="radio" class="nes-radio" name="answer" />
-        <span>No</span>
-      </label>`)
+stories.add('input.radio', () => {
+  const isDark = boolean('is-dark', false) ? 'is-dark' : '';
+
+  return (`
+    <label>
+      <input type="radio" class="nes-radio ${isDark}" name="answer" checked />
+      <span>Yes</span>
+    </label> <label>
```

Comment:
Could you start a new line here. ↩️ 

```suggestion
    </label>
    <label>
```

Reply:
![image](https://user-images.githubusercontent.com/22016005/53414330-3ef6d500-39ad-11e9-8097-860b69f4164b.png)